src/hacking/building-for-openharmony.md

```diff
@@ -4,189 +4,29 @@
 Support for OpenHarmony is currently in-progress and these instructions might change from time to time and might also be incomplete.
 </div>
 
-## Get the OpenHarmony tools
+The building for servo is currently done in two steps and needs a Linux workstation and MacOS/Windows computer for bundling.
+1. Build the shared library (libservo.so).
+2. Create the App Bundle.
```

Comment:
That's not quite correct. For OpenHarmony I think everything can be done completely on Linux. For HarmonyOS generating the signature file is easiest with DevEco (windows / mac), but I do think there is an official guide on how to create a signature file manually. 

Once you have the signature file (and configured your Linux workstation correctly), `mach build --ohos` will both build libservoshell.so and additionally bundle it into a signed hap file, which you can then install via `./mach install --ohos [--release --flavor=...]`